spec/model/topic_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require_relative '../../app/model/topic'
+
+describe Topic do
+
+  it 'should return all Topics' do
+    topics = Topic.all(base_path: 'spec/')
```

Comment:
override for testing base_path: https://relishapp.com/rspec/rspec-mocks/v/2-14/docs/method-stubs/allow-with-a-simple-return-value
best would be in a central place, so you don't have do care about in every spec